### PR TITLE
328 UI adjust the warning message not to overlay login.gov button

### LINKED ISSUE #327: Clean-up homepage text
# 🐛 Summary #

The Warning message on the home page has duplication with words within the text. 

## To reproduce ##

1. Go to https://crossfeed.cyber.dhs.gov/
2. Read the Warning

## Expected behavior ##

No duplicated words should be present

## Any helpful log output or screenshots ##

Paste the results here:

Screenshot 2024-06-06 at 10 23 00 AM
Screenshot 2024-06-06 at 10 23 00 AM

### LINKED ISSUE #328: UI: Adjust the Warning message not to overlay LOGIN.GOV button
# 🐛 Summary #

When using a tablet or similar device with small screen resolution,  the Warning message at the bottom overlays on the LOGIN.COM button. This then blocks the User the ability to log in.

## To reproduce ##

1. Go to https://crossfeed.cyber.dhs.gov/
1. Then resize browser to a smaller size

## Expected behavior ##

We need the page to be more responsive for the User always able to access the LOGIN.GOV button.

Screenshot 2024-06-06 at 10 34 35 AM

### PR DESCRIPTION
**Pull request by hawkishpolicy**
#  #
Adjusted warning message on homepage to not overlap login.gov button
## 🗣 Description ##
- removed grid layout and absolute positioning of warning banner
- implemented a simple flex column with space-between
- removed duplicate verbiage and typos in Warning Message text

## 💭 Motivation and context ##
- improves user experience
- allows mobile users to access the site
- closes #327 closes #328 

## 📷 Screenshots (if appropriate) ##
Screenshot 2024-06-06 at 12 20 55 PM
Screenshot 2024-06-06 at 12 21 06 PM
Screenshot 2024-06-06 at 12 21 14 PM

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by schmelz21 — APPROVED** [review]
LGTM

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Matthew-Grayson) [inline .vscode/settings.json]: Remove this file from the PR.

File: frontend/src/components/WarningBanner/Warning.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export const CrossfeedWarning: React.FC = (props) => {
     borderWidth: 2,
     padding: theme.spacing(1),
     backgroundColor: 'inherit',
-    position: 'absolute',
+    // position: 'absolute',
```
Comment (Matthew-Grayson) [inline frontend/src/components/WarningBanner/Warning.tsx]: Delete this instead of commenting it out.

**Review by Matthew-Grayson — CHANGES_REQUESTED** [review]
Works great!

File: frontend/src/components/WarningBanner/Warning.tsx (hunk 2 above)
Comment (hawkishpolicy) [reply frontend/src/components/WarningBanner/Warning.tsx]: i thought i did..lol

**Review by Matthew-Grayson — APPROVED** [review]

**Pull request merged**